e2e/internal/e2e/image.go

```diff
@@ -50,3 +53,28 @@ func EnsureImage(t *testing.T) {
 		t.Fatalf("Unexpected failure: %+v", err)
 	}
 }
+
+// MakeTmpDir will make a tmp dir and return a string of the path, and a full
+// path (with a container name 'test_container.sif').
+func MakeTmpDir(t *testing.T) (string, string) {
+	name, err := ioutil.TempDir("", "stest.")
```

Comment:
for e2e tests there's an environment variable that specifies which temporary directory to use for the test. All the temporary files should be created under that directory, so that it's easy to clean up.

Reply:
Okay, I didn't know that. I'll fix this...